src/components/vm/filesystems/vmFilesystemsCard.jsx

```diff
@@ -28,7 +28,7 @@ import {
     List, ListItem,
     Modal,
     Popover,
-    Radio, Text, TextContent, TextInput, TextVariants, Tooltip
```

Comment:
This "Stop providing options for shaed memory when creating shared directories" commit has a typo (not important), but refers to #350 which is *this* PR. I think it wants to refer to #343 and #344 instead, and having correct issue links in changelogs is fairly useful/important IMHO.

Reply:
Fixed.